### PR TITLE
Address Getting Started review

### LINKED ISSUE #22500: Correct Getting Started guide
### Description

QE has reviewed this guide

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #22500

@vaceksimon Can you verify this look good now?

### CODE REVIEW

File: docs/guides/getting-started/templates/first-app.adoc [hunk 1]
```diff
@@ -3,19 +3,18 @@
...
 . Confirm that *Standard flow* is enabled.
-. Click *Save*.
-
+. Click *Next*.
++
 image::add-client-1.png[Add Client]
 
-After the client is created, make these updates to the client:
-
 . Scroll down to *Access settings*.
```
Comment (vaceksimon) [inline docs/guides/getting-started/templates/first-app.adoc:18]: I do not think you can scroll down to any other section here.

**Review by vaceksimon — CHANGES_REQUESTED** [review]
Looks very good, I appreciate the changes.

I just noticed that it may be worth it to change the picture [add-client-2.png](https://github.com/keycloak/keycloak/blob/main/docs/guides/images/add-client-2.png) to the [version that matches the current state](https://imgur.com/a/X9Y0MGW).

I would also suggest to emphasize the admin username and password and explicitly state them in the text in the [openshift section](https://github.com/keycloak/keycloak/blob/main/docs/guides/getting-started/getting-started-openshift.adoc#start-keycloak) of the guide, just below the step 1.

Otherwise good work!

**Comment by andymunro** [comment]
Thanks, @vaceksimon for the corrections. Let me know if this work or if you find anything else.

**Review by vaceksimon — CHANGES_REQUESTED** [review]
Thank you for such a quick fix. I have one last little suggestion after which this PR can be merged. Great work!

**Review by vaceksimon — APPROVED** [review]
LGTM!

**Review by ahus1 — APPROVED** [review]
Thank you for this contribution!

**Pull request merged**